vividus-plugin-web-app-to-rest-api/src/test/java/org/vividus/bdd/steps/integration/ResourceCheckStepsTests.java

```diff
@@ -127,8 +127,8 @@
           + "<title>Title of the document</title>\r\n"
           + "</head>\r\n"
           + "<body>\r\n"
-          + "  <a id='about' href='https://vividus.org/about'>About</a>\r\n"
-          + "  <video id='about'>Some video without attributes</a>\r\n"
+          + "  <a id='link-id' href='https://vividus.org/about'>About</a>\r\n"
+          + "  <video id='video-id'>Some video without attributes</a>\r\n"
```

Comment:
The root cause of the change is
https://github.com/jhy/jsoup/commit/7cf9d03bf311494df652a05c2528eb06b2373f9d